casts/row/src/UILink/UILink.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { ColorVariants, UILabel, TypographyVariants } from '@tonlabs/uikit.themes';
+import { TouchableOpacity } from '@tonlabs/uikit.controls';
+import { UILayoutConstant, UISkeleton } from '@tonlabs/uikit.layout';
+import type { UILinkProps } from './types';
+import { Logo } from './Logo';
+import { Icon } from './Icon';
+
+export const UILink: React.FC<UILinkProps> = ({
+    title,
+    description,
+    logoSource,
+    iconAfterTitleSource,
+    onPress,
+    loading,
+    testID,
+}: UILinkProps) => {
+    return (
+        <UISkeleton show={!!loading}>
```

Comment:
I actually thought that skeleton will be around logo, title and subtitle, not around whole row. Like the point of skeletons is to show what layout will be after it's loaded

Reply:
I have separated the logo from the text block
![image](https://user-images.githubusercontent.com/36547799/143434370-ba2f3479-afd7-4fb9-93ba-bc726617587d.png)